Formula/haskell-language-server.rb

```diff
@@ -0,0 +1,44 @@
+class HaskellLanguageServer < Formula
+  desc "Integration point for ghcide and haskell-ide-engine. One IDE to rule them all"
+  homepage "https://github.com/haskell/haskell-language-server"
+
+  url "https://github.com/haskell/haskell-language-server.git",
+    tag:      "1.0.0",
+    revision: "4cd1cf934638881e52b3eba9f70157a4b799c0e9"
+
+  license "Apache-2.0"
+
+  head "https://github.com/haskell/haskell-language-server.git"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  pour_bottle? do
+    reason "Need help with that ^^'"
+    satisfy { false }
+  end
+
+  depends_on "cabal-install" => :build
+  depends_on "ghc" => :build
+
+  def install
+    system "cabal", "v2-update"
+    # taken from haskell-language-server/install/src/Cabal.hs
+    system "cabal", "v2-install",
+      "exe:haskell-language-server",
+      "exe:haskell-language-server-wrapper",
+      "-w", "ghc",
+      "--write-ghc-environment-files=never",
+      "--max-backjumps=5000",
+      "--overwrite-policy=always",
+      "--project-file=cabal.project",
+      *std_cabal_v2_args
+  end
+
+  test do
+    (testpath/"test.hs").write("f :: Int -> Int\nf x = x + 1")
+    system bin/"haskell-language-server-wrapper", testpath/"test.hs"
```

Comment:
Can we do an `assert_match` or `assert_equal` to test the output of this?

Reply:
Will try.

Reply:
Seems it is now better :)